### ISSUE TITLE
Cannot find module puppeteer

### ISSUE THREAD

**Issue by Tragio** [harlan-zw/unlighthouse#16]
Hi @harlan-zw 👋 

As you can see in the image unlighthouse never picks puppeteer, even though it is installed globally. I'm using [nvm](https://github.com/nvm-sh/nvm) and the path would be `/Users/tragio/.nvm/versions/node/v14.19.1/lib/node_modules/` and not `file:///Users/tragio, file:///Users/tragio/node_modules` as stated in the error message 🤔 I wonder if the error is related to that.

![CleanShot 2022-04-21 at 09 14 49](https://user-images.githubusercontent.com/5109593/164551413-cd7a276b-4300-4252-9d33-220dced0321a.png)

Thank you for your amazing work 🙏

**Comment by zuramai** [comment]
Same here

**Comment by harlan-zw** [comment]
Hey @Tragio, @zuramai, sorry you had issues with the package.

I've made a binary that depends on puppeteer to try to get around this issue. Could someone try the below:


Using pnpm

```bash
pnpm dlx unlighthouse --site <your-site>
# or if that doesn't work try:
# pnpm dlx unlighthouse-puppeteer --site example.com
```

Using npx

```bash
npx unlighthouse --site <your-site>
# or if that doesn't work try:
# npx unlighthouse-puppeteer --site example.com
```

**Comment by Tragio** [comment]
@harlan-zw I tried to use but still had the same error `Failed to find a chrome / chromium binary to run. Add the puppeteer dependency to your project to resolve. Error: Cannot find module puppeteer imported from file:///Users/tragio, file:///Users/tragio/node_modules`

**Comment by lasfito** [comment]
Hi, 
I'm getting the very same error even after trying these alternatives

**Comment by Tragio** [comment]
@harlan-zw does this help?

![CleanShot 2023-03-02 at 13 39 51@2x](https://user-images.githubusercontent.com/5109593/222444669-1bafbbf2-d1d4-4af2-b1c3-6612f5a7a3da.jpg)

**Comment by LachlanArthur** [comment]
This might be caused by a different channel of Chrome being installed, so puppeteer cannot find it.

In my case, I have Chrome Dev installed.

I fixed the issue by defining the path to the Chrome executable in the unlighthouse config (your Chrome path may be different to this):

```
export default {
	puppeteerOptions: {
		executablePath: "C:\\Program Files (x86)\\Google\\Chrome Dev\\Application\\chrome.exe"
	}
}
```

I've run into this problem on similar projects, so I've opened a PR on the [`chrome-launcher`](https://github.com/GoogleChrome/chrome-launcher/pull/297) library to add detection for Beta and Dev channels.
